ruddock/modules/rotation/routes.py

```diff
@@ -44,3 +50,34 @@ def update_comment(prefrosh_id):
   helpers.update_comments(prefrosh_id, flask.request.form.get("comments", ""))
   helpers.update_votes(prefrosh_id, flask.request.form)
   return flask.redirect(flask.url_for("rotation.show_prefrosh", prefrosh_id=prefrosh_id))
+
+@blueprint.route('/move/', methods=['POST'])
```

Comment:
I think the convention has been to omit the trailing slash (e.g. `/foo` instead of `/foo/`).